system/jlib/jstats.cpp

```diff
@@ -1862,6 +1868,24 @@ class CStatisticCollection : public CInterfaceOf<IStatisticCollection>
         }
         return false;
     }
+    virtual bool setStatistic(const char *scope, StatisticKind kind, unsigned __int64 & value) override
```

Comment:
Doesn't really need to be a member function - could be implemented using the existing public interface.

Reply:
It is used from workunit.cpp by the aggregate loader.  It could be done using the existing member functions but it would be cumbersome to add code to the aggregate loader to split the scopes and navigate down the tree.

Reply:
I agree it should be in a function.  The comment is that it could be a global function - which would reduce the number of functions in the interface.  Debatable style wise, but was connected to the large number of new functions in the interface.